src/odin/maneuver.cc

```diff
@@ -764,6 +766,36 @@ void Maneuver::set_contains_obvious_maneuver(bool contains_obvious_maneuver) {
   contains_obvious_maneuver_ = contains_obvious_maneuver;
 }
 
+bool Maneuver::has_combined_enter_exit_roundabout() const {
+  return has_combined_enter_exit_roundabout_;
+}
+
+void Maneuver::set_has_combined_enter_exit_roundabout(bool has_combined_enter_exit_roundabout) {
+  has_combined_enter_exit_roundabout_ = has_combined_enter_exit_roundabout;
+}
+
+float Maneuver::roundabout_length(const Options::Units& units) const {
```

Comment:
How is this length different from the maneuver length?

Reply:
For a combined enter/exit roundabout maneuver - this is the length of the circle. The maneuver length is the length of the circle and the length after the circle